suite-common/message-system/schema/config.schema.v1.json

```diff
@@ -346,6 +167,55 @@
                 },
                 "additionalProperties": false
             }
+        },
+        "experiments": {
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Experiments",
+                "type": "object",
+                "required": ["conditions", "experiment"],
+                "properties": {
+                    "conditions": {
+                        "$ref": "#/definitions/conditions"
+                    },
+                    "experiment": {
+                        "title": "Experiments item",
+                        "description": "Used for AB testing",
+                        "type": "object",
+                        "required": ["id", "groups"],
+                        "properties": {
+                            "id": {
+                                "type": "string"
+                            },
+                            "groups": {
+                                "type": "array",
+                                "minItems": 2,
```

Comment:
I think not needed

Reply:
It doesn't make sense to have an experiment with zero or one item. What do you think?

Reply:
And how we will handle the experiments that have the clear winner? I guess we would need to keep that experiment for older Suite versions with only one group and 100%. We can't just remove the experiment if the winner is not the default option.

Reply:
Group A - 100, Group B - 0